lib/components/form/call-taker/advanced-options.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jsx-a11y/label-has-for */
```

Comment:
Can you add a FIXME for this file-wide disable?

Reply:
I added the requested FIXMEs for rule exceptions in https://github.com/opentripplanner/otp-react-redux/pull/390/commits/1a526dd2886f67b837dfe118666f9d9d719988f5.